bin/fit_posparams

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Analysis script comparing internally-tracked (theta, phi) to fvc-measured (x,y).
+The goal is to examine the general performance of all positioners during any
+move (not just a specialized calibration sequence).
+
+Data sources:
+
+    theta and phi:
+    --------------
+    The positioner moves database contains time-stamped data for values POS_T
+    and POS_P. These correspond to focalplane coordinates posintTP.
+    
+    x and y:
+    --------
+    The NERSC and KPNO servers contain numerous FITS files. These are processed
+    by desimeter. A result X_FP, Y_FP is generated from the centroids. This
+    corresponds to focalplane coordinates ptlXY.
+    
+    As of 2020-04-08, J. Guy identified 9472 such files from the commissioning
+    operations so far at the Mayall, and has run them through desimeter. See
+    Julien's email to desi-commiss@desi.lbl.gov of 2020-04-08, in which he
+    attaches several descriptive slides.
+
+    The fvc catalogs he produced are on NERSC at:
+        /global/cfs/cdirs/desi/engineering/fvc/desimeter/v0.2.0/
+
+    And the matched coordinates per positioner are on NERSC at:
+        /global/cfs/cdirs/desi/engineering/fvc/desimeter/v0.2.0/positioners/
+"""
+
+import os
+import sys
+import cProfile, pstats
+import pandas
+import math
+import numpy
+import time
+import datetime
+import multiprocessing
+import yaml
+
+# DESI-specific imports
+# path handling is to be improved as I migrate better into github/desimeter
+desimeter_path = os.path.abspath('../py/desimeter/')
+sys.path.append(desimeter_path)
+import posparams.fitter as fitter
+import posparams.plotter as plotter
+sys.path.append(fitter.petal_path)
+import petaltransforms
```

Comment:
Let's discuss this petaltransforms dependency.  So far desimeter has purposefully avoided any dependencies on either online or offline code so that it could be used by either.  At minimum we need to support this without manipulating `sys.path` within the code, but we should discuss this in the broader context of desimeter dependencies.

Reply:
Yeah, also petaltransforms is a bit ugly.
1. I definitely intend to switch over to desimeter.transform.ptl2fp.
2. There is also the more interesting postransforms module, which converts between (t,p) and (x,y). These are not simple transforms. I am debating in my mind forking these conversions into desimeter, in a clean new desimeter-specific module. Would like to discuss pros / cons with you.